include/Scoreboard.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef SCOREBOARD_HPP_
+#define SCOREBOARD_HPP_
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <cstring>
+#include <vector>
+#include <algorithm>
+
+class PlayerData
```

Comment:
since all members are public, it would be a good idea to replace this `class` by a `struct` and removing the `public:` label.